src/sanitizer.js

```diff
@@ -198,11 +198,16 @@ export function sanitizeHtml(html) {
         } else {
           attribs = scrubbed.attribs;
           // Restore some of the attributes that AMP is directly responsible
-          // for, such as "on".
+          // for, such as "on" or amp-bind's bracketed attributes.
           for (let i = 0; i < attribs.length; i += 2) {
-            if (WHITELISTED_ATTRS.indexOf(attribs[i]) != -1) {
+            const attrib = attribs[i];
+            if (WHITELISTED_ATTRS.indexOf(attrib) != -1) {
               attribs[i + 1] = savedAttribs[i + 1];
-            } else if (attribs[i].search(WHITELISTED_ATTR_PREFIX_REGEX) == 0) {
+            } else if (attrib.search(WHITELISTED_ATTR_PREFIX_REGEX) == 0) {
+              attribs[i + 1] = savedAttribs[i + 1];
+            } else if (attrib[0] == '[' && attrib[attrib.length - 1] == ']') {
+              // Completely restore bind attribute values, as sanitizer does not recognize
```

Comment:
Allow me to answer a little out of order here. As far as bind is concerned, when bind parses 

`<a [onclick]="alert(document.cookie)"></a>`

it will see that onclick is not one of the whitelisted tags in bind-validator.js and will ignore it. For this, I don't think it's any worse than a user adding `[onclick]` as an attribute on an element outside of a mustache template while bind is enabled. 

As far as other components are concerned, besides bind, which components depend on this for sanitzation? Is there a risk that they will see [onclick] and change it to onclick? 

We can definitely re-do the whitelist check; the problem I see with that is the whitelist caja uses will need to be updated to include all attributes we're going to allow binding to., further modifying this code from the original. That's definitely a possibility, but it seems very error prone if the whitelisted attributes in bind and the whitelisted attributes here start to drift apart. What are your thoughts on this?